### ISSUE TITLE
`Unable to save plugin settings` error

### ISSUE THREAD

**Issue by remal** [jshiell/checkstyle-idea#574]
I updated the plugin to version `5.67.0` and started getting this error message: `Unable to save plugin settings`.

<details>
  <summary>Corresponding exception in IDEA logs</summary>

```
2022-06-04 21:09:01,233 [ 310323]   WARN - #c.i.o.c.i.s.StoreUtil - Save settings failed
java.lang.Exception: Cannot get CheckStyle-IDEA-Module component state
	at com.intellij.configurationStore.ComponentStoreImpl.commitComponents$intellij_platform_configurationStore_impl(ComponentStoreImpl.kt:244)
	at com.intellij.configurationStore.ProjectWithModulesStoreImpl.commitModuleComponents(ProjectStoreImpl.kt:193)
	at com.intellij.configurationStore.ProjectWithModulesStoreImpl.access$commitModuleComponents(ProjectStoreImpl.kt:161)
	at com.intellij.configurationStore.ProjectWithModulesStoreImpl$saveModules$$inlined$withEdtContext$intellij_platform_configurationStore_impl$1.invokeSuspend(ComponentStoreImpl.kt:730)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.intellij.openapi.application.constraints.BaseConstrainedExecution$Companion$scheduleWithinConstraints$1.invoke(BaseConstrainedExecution.kt:68)
	at com.intellij.openapi.application.constraints.BaseConstrainedExecution$Companion.scheduleWithinConstraints(BaseConstrainedExecution.kt:71)
	at com.intellij.openapi.application.constraints.BaseConstrainedExecution.scheduleWithinConstraints(BaseConstrainedExecution.kt:38)
	at com.intellij.openapi.application.impl.BaseExpirableExecutorMixinImpl.access$scheduleWithinConstraints$s1153900543(BaseExpirableExecutorMixinImpl.kt:12)
	at com.intellij.openapi.application.impl.BaseExpirableExecutorMixinImpl$scheduleWithinConstraints$$inlined$Runnable$1.run(Runnable.kt:19)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:215)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:197)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:873)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:511)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:69)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:112)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:42)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:898)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:439)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:803)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:438)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:604)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:436)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:873)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:484)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.NullPointerException
	at java.base/java.util.ArrayList.<init>(ArrayList.java:179)
	at org.infernus.idea.checkstyle.config.ModuleConfigurationState$ModuleSettings.setActiveLocationIds(ModuleConfigurationState.java:132)
	at org.infernus.idea.checkstyle.config.ModuleConfigurationState.getState(ModuleConfigurationState.java:75)
	at org.infernus.idea.checkstyle.config.ModuleConfigurationState.getState(ModuleConfigurationState.java:20)
	at com.intellij.configurationStore.ComponentStoreImpl.commitComponent(ComponentStoreImpl.kt:334)
	at com.intellij.configurationStore.ComponentStoreImpl.commitComponents$intellij_platform_configurationStore_impl(ComponentStoreImpl.kt:240)
	... 41 more
```
</details>

<details>
  <summary>My <code>.idea/checkstyle-idea.xml</code> file</summary>

```
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="CheckStyle-IDEA" serialisationVersion="2">
    <checkstyleVersion>10.3</checkstyleVersion>
    <scanScope>JavaOnlyWithTests</scanScope>
    <suppressErrors>true</suppressErrors>
    <copyLibs>true</copyLibs>
    <option name="thirdPartyClasspath" />
    <option name="activeLocationIds">
      <option value="1c747537-eab4-4d99-8ecc-35b14051ccc6" />
    </option>
    <option name="locations">
      <list>
        <ConfigurationLocation id="56b5990a-2009-4a76-a4f2-55b40175b6b8" type="BUNDLED" scope="All" description="Sun Checks">(bundled)</ConfigurationLocation>
        <ConfigurationLocation id="8b1465fb-94a4-4679-bb26-e6d97826bf16" type="BUNDLED" scope="All" description="Google Checks">(bundled)</ConfigurationLocation>
        <ConfigurationLocation id="1c747537-eab4-4d99-8ecc-35b14051ccc6" type="PROJECT_RELATIVE" scope="All" description="Project checkstyle">$PROJECT_DIR$/checkstyle.xml</ConfigurationLocation>
      </list>
    </option>
  </component>
</project>
```
</details>

**Comment by mmoayyed** [comment]
I am also seeing the same thing with checkstyle 10.3. 

IDEA:

```
IntelliJ IDEA 2022.1.2 (Ultimate Edition)
Build #IU-221.5787.30, built on May 31, 2022
Runtime version: 11.0.15+10-b2043.56 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.6.6
GC: G1 Young Generation, G1 Old Generation
Memory: 5000M
Cores: 8
CheckStyle-IDEA (5.67.0)

Kotlin: 221-1.6.21-release-337-IJ5787.30
```

Checkstyle settings such as the suppression file or the import file cannot be saved. Once the settings dialog is closed, the values always remain as empty for those fields.

<img width="497" alt="image" src="https://user-images.githubusercontent.com/1205228/172046740-b4ae3df9-4bd0-4bbe-9792-791be2970cf9.png">

**Comment by jshiell** [comment]
Thanks both for the reports - NPE should be fixed in 5.67.1. The properties is a more embarassing problem, covered in #573.
